Extension/src/nativeStrings.json

```diff
@@ -177,6 +177,9 @@
         "text": "+%d specializations",
         "hint": "Refers to C++ template specializations. This is part of a description indicating there are N additional specializations of a function.  %d is replaced with that number."
     },
+    "compiler": {
+        "text": "Note: IntelliSense is not fully configured. Use the Select Default Compiler command to finish configuration."
```

Comment:
I believe this should be `'Select Default Compiler' command` with the `'` added since that's how VS Code's references commands in the settings UI.